examples/desktop/mapbook.xml

```diff
@@ -4,6 +4,10 @@
         The mapping services define the source of the mapping data.
     -->
 
+    <map-source name="usng-grid" type="usng">
+        <layer name="usng-grid" status="on" />
```

Comment:
It may make sense to turn this off before merging as USNG will not really be globally applicable.